db/seeds.d/108-subcription-bookmarks.rb

```diff
@@ -0,0 +1,17 @@
+# This file should contain all the record creation needed to seed the database with its default values.
+# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
+#
+# !!! PLEASE KEEP THIS SCRIPT IDEMPOTENT !!!
+#
+
+Bookmark.without_auditing do
```

Comment:
Added creation of two bookmarks here for the user that lists hyprvisors and list guests. Feel free to suggest better names for the bookmarks.